src/dmd/backend/dvec.d

```diff
@@ -0,0 +1,605 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
```

Comment:
I suggest adding a description of the module. I have no idea what the code in this file does. Also, a more descriptive file name wouldn't hurt